pkg/sdk/resource_monitors.go

```diff
@@ -24,17 +29,138 @@ type resourceMonitors struct {
 }
 
 type ResourceMonitor struct {
-	Name string
+	Name                     string
+	CreditQuota              float64
+	UsedCredits              float64
+	RemainingCredits         float64
+	Frequency                Frequency
+	StartTime                string
+	EndTime                  string
+	SuspendTriggers          []TriggerDefinition
```

Comment:
it may make sense to have this just be an int. i.e. `SuspendAt *int`

Reply:
Ok, I guess I didn't want to change the behaviour. I thought `suspend_triggers` still might be used under some circumstances.
So what should we do about the `suspend_triggers` argument then?
I think we can:
1) just get rid of it
2) ignore it and just take the value from `suspend_trigger`
3) use the trigger with the lowest threshold from `suspendTrigger` and `suspend_triggers`

I think we should either remove it or at least explain the changed semantics. Otherwise it might be confusing for the users.

Reply:
suspend_triggers is a deprecated attribute since it never makes sense to have more than one suspend trigger. We have to continue to support it until the next major provider release (1.0). What i would do is just choose the lowest value in the list and use that

Reply:
ok